pkgs/misc/cups/drivers/ctzpos-cups/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, cups, gcc, rpmextract }:
+
+stdenv.mkDerivation rec {
+  pname = "ctzpos-cups";
+  version = "1.2.4";
+
+  src = fetchurl {
+    url = "https://www.citizen-systems.co.jp/english/support/download/printer/driver/cups/data_cups/${version}.0/ctzpos-cups-${version}-0.src.rpm";
+    sha256 = "0lpligmhvvr0j39f5lc8f7y44zjry95d681wm65jw76alavfn1wa";
```

Comment:
```suggestion
    sha256 = "0l0wnmhn98jidziv1w8g1jwagb109rm5zl3qlv8wg6xch9z6cpjb";
```

Hash changed.

Reply:
apparently it changed again (now it seems to be `05f6flh9m99dnbkzz3jvkva6nylj5zda25yi21wnd7pyd6d23qd7`)...